src/tools/cephfs/top/cephfs-top

```diff
@@ -203,7 +225,64 @@ class FSTop(object):
             # If the terminal do not support the visibility
             # requested it will raise an exception
             pass
-        self.run_display()
+        fs_list = self.get_fs_names()
+        self.run_all_display(fs_list)
+
+    def display_menu(self, stdscr, selected_row_idx):
+        stdscr.clear()
+        h, w = stdscr.getmaxyx()
+        if not fs_list:
+            title = ['No filesystem available',
+                     'Press "q" to go back to home (all filesystem info) screen']
+            pos_x1 = w // 2 - len(title[0]) // 2
+            pos_x2 = w // 2 - len(title[1]) // 2
+            stdscr.addstr(1, pos_x1, title[0], curses.A_STANDOUT | curses.A_BOLD)
+            stdscr.addstr(3, pos_x2, title[1])
+        else:
+            title = ['Filesystems', 'Press "q" to go back to home (all filesystem info) screen']
+            pos_x1 = w // 2 - len(title[0]) // 2
+            pos_x2 = w // 2 - len(title[1]) // 2
+            stdscr.addstr(1, pos_x1, title[0], curses.A_STANDOUT | curses.A_BOLD)
+            stdscr.addstr(3, pos_x2, title[1])
+            for index, name in enumerate(fs_list):
+                x = w // 2 - len(name) // 2
+                y = h // 2 - len(fs_list) // 2 + index
+                if index == selected_row_idx:
+                    stdscr.attron(curses.color_pair(1))
+                    stdscr.addstr(y, x, name)
+                    stdscr.attroff(curses.color_pair(1))
+                else:
+                    stdscr.addstr(y, x, name)
+        stdscr.refresh()
+
+    def set_key(self, stdscr):
+        curses.curs_set(0)
+        curses.init_pair(1, curses.COLOR_BLACK, curses.COLOR_WHITE)
+        curr_row = 0
+        self.display_menu(stdscr, curr_row)
+
+        while 1:
+            key = stdscr.getch()
+
+            if key == curses.KEY_UP and curr_row > 0:
+                curr_row -= 1
+            elif key == curses.KEY_DOWN and curr_row < len(fs_list) - 1:
+                curr_row += 1
+            elif (key == curses.KEY_ENTER or key in [10, 13]) and fs_list:
```

Comment:
I think enter or space for selection would be better.

Reply:
Do you really think we should also allow space bar for selection?

Reply:
Yeah that's what I meant. But checking on top(1) today, it's just the Enter key that commits. So please ignore this comment. May be we could use space bar for multiple `fs` selections, but that would be a later req.